ocaml/xapi/updateinfo.ml

```diff
@@ -405,6 +405,21 @@ module LivePatch = struct
          )
 end
 
+module Severity = struct
+  type t = None | High
+
+  let to_string = function None -> "None" | High -> "High"
+
+  let of_string = function
```

Comment:
I would use `match  String.( x |> lowercase_ascii |> trim) with "none" -> .. | "high" -> ..` to be a bit more flexible how these are encoded. But it's fine to be strict, too.

Reply:
Thank you